scripts/iso-manager.sh

```diff
@@ -1,11 +1,20 @@
 #!/bin/bash
+set -e
+set -u
+# The trickery below is to only run `set -o pipefail`, which is a Bash'ism,
```

Comment:
Do you need that trick?

I mean, your shebang is explicitly requiring Bash, so why bother?

Reply:
we have this in the bootstrap script too I guess the shebang is not enough ?

Reply:
In theory someone could replace `/bin/bash` by a symlink to another shell, but that's asking for trouble.
Other than that case, I think we are safe, but I may be missing something.

I just want to avoid Cargo cult programming (see https://en.wikipedia.org/wiki/Cargo_cult_programming)

> Cargo cult programming is a style of computer programming characterized by the ritual inclusion of code or program structures that serve no real purpose.